doc/psidk/src/development/services/rust-service/README.md

```diff
@@ -8,3 +8,4 @@
 - [Minimal UI](minimal-ui.md)
 - [Tables](tables.md)
 - [GraphQL](graphql.md)
+- [Pre-action Initialization](pre-action.md)
```

Comment:
You named this document, "Initializing Services" in the main `SUMMARY.md`. Don't care which, but should probably be consistent

Reply:
@cool-ant commenting to make sure you saw this